source/_docs/eu-region.md

```diff
@@ -9,59 +9,119 @@ earlyaccess: true
+Pantheon is extending early access to contract organizations (Enterprise, Reseller, OEM, and EDU+) that want to take a new site live from the European Union (**EU**) region.
 
-### Details
+## Enable Early Access
+Participation in Early Access is opt-in and available to contract organizations only. To learn more about enabling this feature, contact your dedicated account manager or our [Sales team](https://pantheon.io/contact-us){.external}.
 
-Site resources, including the database container, are provisioned in the European Union. Traffic is served through the Global CDN, with the Amsterdam point of presence as origin shield.
+## Use Cases
+There could be many scenarios for wanting to host a site within the EU region rather than the default US. Common use cases include:  
 
-## How to Enable Early Access and Create Sites in the EU Region
+* Compliance standards that require data residency within the borders of the European Union
+* Improve performance and user experience for authenticated traffic originating in or near the European Union
 
-During this feature's Early Access period, participation is opt-in only for contract customers. All other new sites will continue to be deployed to the default US region. [Give us a call](https://pantheon.io/contact-us){.external} for information about pricing and how to enable your account for early access to the EU Region.
 
-Once granted early access, our Support team will create the new site for you.
+## Region Availability
+Once enabled, this organization feature allows <a href="/docs/change-management/#organizations-roles-and-permissions" data-proofer-ignore>privileged users</a> to designate one of two available regions at the time of site creation:
 
-## Use Terminus to Display Sites and Regions
+* US (Default)
+* EU
 
-In the following examples, replace or assign `$SITE` with your site name or UUID.
+Site resources, including the database container, are provisioned in the specified region. For EU sites, traffic is served through the Global CDN, with the Amsterdam point of presence as origin shield.
 
-To display a specific site's plan and region information:
+
+## Go Live in the EU
+### Create a New Site (Required)
+
+1. Install and authenticate [Terminus](/docs/terminus). If you're already running Terminus, be sure and update to the [latest version](/docs/terminus/updates/) (2.0 minimum required).
+2. Use Terminus to create a new site associated with your organization and include the `--region=eu` option.  
+
+ For example (replace `my-eu-site-name`, `My EU Site Name`, `WordPress` and `My Organization Name` accordingly):
+
+  ```bash
+  terminus site:create my-eu-site-name "My EU Site Name" "WordPress" --org "My Organization Name" --region eu
+  ```
+
+  ![terminus site:create my-eu-site "My EU Site" "WordPress" --org "Rachel Pantheor" --region eu](/source/docs/assets/images/create-site-eu.png)
+
+### Review Site Region (Optional)
```

Comment:
Let's move this step out of the "Go Live" flow. Maybe to it's own separate section?